src/components/FilterPanel.tsx

```diff
@@ -92,11 +79,11 @@ export default function FilterPanel({
     [t('minAmount')]: {
       comp: (
         <MinAmountFilter
-          min={minAmount ?? 0}
```

Comment:
I'm curious why this was like this before... You'd think with `undefined` it'd default to 0 or something 🤔

Reply:
It's so the console wouldn't complain about changing an uncontrolled component.